src/i18n/hi/ui.ts

```diff
@@ -1,3 +1,58 @@
+	'site.title': 'Astro दस्तावेज़ीकरण', // "Astro Documentation"
+	'site.description': 'कम Client-side Javascript के साथ तेज़ वेबसाइटें बनाएं।', // "Build faster websites with less client-side JavaScript."
+	'site.og.imageSrc': '/default-og-image.png?v=1',
+	'site.og.imageAlt':
+		'एक तारों भरे अंतरिक्ष में Astro लोगो, जिसमें दाएँ से एक जामुनी शनि ग्रह की तरह ग्रह तैरता है', // "astro logo on a starry expanse of space, with a purple saturn-like planet floating in the right foreground"
+	// Left Sidebar
+	'leftSidebar.a11yTitle': 'प्रमुख', // "Primary"
+	'leftSidebar.learnTab': 'सीखें', // "Learn"
+	'leftSidebar.referenceTab': 'संदर्भ', // "Reference"
+	'leftSidebar.viewInEnglish': 'अंग्रेज़ी में देखें', // "View in English"
+	'leftSidebar.sponsoredBy': 'समर्थित करने वाला', // "Sponsored by"
+	// Right Sidebar
+	'rightSidebar.a11yTitle': 'माध्यमिक', // "Secondary"
+	'rightSidebar.onThisPage': 'इस पृष्ठ पर', // "On this page"
+	'rightSidebar.overview': 'अवलोकन', // "Overview"
+	'rightSidebar.community': 'समुदाय', // "Community"
+	'rightSidebar.joinDiscord': 'हमसे Discord में शामिल हों', // "Join us on Discord"
+	'rightSidebar.readBlog': 'हमारे ब्लॉग पोस्ट पढ़ें', // "Read our blog posts"
+	'rightSidebar.openCollective': 'हमारा ओपन कलेक्टिव', // "Our Open Collective"
+	'rightSidebar.contribute': 'योगदान करें', // "Contribute"
+	'rightSidebar.contributorGuides': 'योगदानकर्ता मार्गदर्शिकाएँ', // "Contributor Guides"
+	'rightSidebar.editPage': 'इस पृष्ठ को संपादित करें', // "Edit this page"
+	'rightSidebar.translatePage': 'इस पृष्ठ को अनुवाद करें', // "Translate this page"
+	'rightSidebar.github': 'एस्ट्रो डॉक्स GitHub पर', // "Astro Docs on GitHub"
+	// Footer
+	'footer.privacyPolicy': 'गोपनीयता नीति', // "Privacy Policy"
+	// `<ThemeToggleButton>` accessibility labels
+	'themeToggle.useLight': 'Light थीम उपयोग करें', // "Use light theme"
+	'themeToggle.useDark': 'Dark थीम उपयोग करें', // "Use dark theme"
+	// Used in previous/next page links at the bottom of pages
+	'articleNav.nextPage': 'अगला पृष्ठ', // "Next Page"
+	'articleNav.prevPage': 'पिछला पृष्ठ', // "Back"
+	// Used in `<Since>`: Added in: v0.24.0 [NEW]
+	'since.addedIn': 'जोड़ा गया:', // "Added in:"
+	'since.new': 'नया', // "New"
+	'since.beta': 'बीटा', // "Beta"
+	// Installation Guide
+	'install.autoTab': 'स्वचालित CLI', // "Automatic CLI"
+	'install.manualTab': 'मैन्युअल सेटअप', // "Manual Setup"
+	// `<DeployGuidesNav>` vocabulary
+	'deploy.sectionTitle': 'पृष्ठ संचालन मार्गदर्शिकाएँ', // "Deployment Guides"
+	'deploy.altSectionTitle': 'अधिक पृष्ठ संचालन मार्गदर्शिकाएँ', // "More Deployment Guides"
+	'deploy.filterLabel': 'डिप्लॉय प्रकार से छाँटें', // "Filter by deploy type"
+	'deploy.ssrTag': 'SSR', // "SSR"
+	'deploy.staticTag': 'Static', // "Static"
+	// CMS Guides vocabulary
+	'cms.navTitle': 'और CMS मार्गदर्शिकाएँ', // "More CMS guides"
+	// Migration Guides vocabulary
+	'migration.navTitle': 'और माइग्रेशन मार्गदर्शिकाएँ', // "More migration guides"
+	// Recipes
+});
```

Comment:
Nicely done, couple of suggestions ⬇️ 
```suggestion
export default UIDictionary({
	'a11y.skipLink': 'इसे छोड़कर कंटेंट पर जाएं', // "Skip to Content"
	'a11y.sectionLink': 'शीर्षक', // "Section titled"
	'navbar.a11yTitle': 'शीर्ष', // "Top"
	// Site settings
	'site.title': 'Astro दस्तावेज़ीकरण', // "Astro Documentation"
	'site.description': 'कम Client-side Javascript के साथ तेज़ वेबसाइटें बनाएं।', // "Build faster websites with less client-side JavaScript."
	'site.og.imageSrc': '/default-og-image.png?v=1',
	'site.og.imageAlt':
		'एक तारों भरे अंतरिक्ष में Astro लोगो, जिसमें दाएँ से एक जामुनी शनि ग्रह की तरह ग्रह तैरता है', // "astro logo on a starry expanse of space, with a purple saturn-like planet floating in the right foreground"
	// Left Sidebar
	'leftSidebar.a11yTitle': 'प्रमुख', // "Primary"
	'leftSidebar.learnTab': 'सीखें', // "Learn"
	'leftSidebar.referenceTab': 'संदर्भ', // "Reference"
	'leftSidebar.viewInEnglish': 'अंग्रेज़ी में देखें', // "View in English"
	'leftSidebar.sponsoredBy': 'समर्थित करने वाला', // "Sponsored by"
	// Right Sidebar
	'rightSidebar.a11yTitle': 'माध्यमिक', // "Secondary"
	'rightSidebar.onThisPage': 'इस पृष्ठ पर', // "On this page"
	'rightSidebar.overview': 'अवलोकन', // "Overview"
	'rightSidebar.community': 'समुदाय', // "Community"
	'rightSidebar.joinDiscord': 'हमसे Discord में शामिल हों', // "Join us on Discord"
	'rightSidebar.readBlog': 'हमारे ब्लॉग पोस्ट पढ़ें', // "Read our blog posts"
	'rightSidebar.openCollective': 'हमारा ओपन कलेक्टिव', // "Our Open Collective"
	'rightSidebar.contribute': 'योगदान करें', // "Contribute"
	'rightSidebar.contributorGuides': 'योगदानकर्ता मार्गदर्शिकाएँ', // "Contributor Guides"
	'rightSidebar.editPage': 'इस पृष्ठ को संपादित करें', // "Edit this page"
	'rightSidebar.translatePage': 'इस पृष्ठ को अनुवाद करें', // "Translate this page"
	'rightSidebar.github': 'Astro Docs देखे GitHub पर', // "Astro Docs on GitHub"
	// Footer
	'footer.privacyPolicy': 'गोपनीयता नीति', // "Privacy Policy"
	// `<ThemeToggleButton>` accessibility labels
	'themeToggle.useLight': 'Light थीम उपयोग करें', // "Use light theme"
	'themeToggle.useDark': 'Dark थीम उपयोग करें', // "Use dark theme"
	// Used in previous/next page links at the bottom of pages
	'articleNav.nextPage': 'अगला पृष्ठ', // "Next Page"
	'articleNav.prevPage': 'पिछला पृष्ठ', // "Back"
	// Used in `<Since>`: Added in: v0.24.0 [NEW]
	'since.addedIn': 'जोड़ा गया:', // "Added in:"
	'since.new': 'नया', // "New"
	'since.beta': 'बीटा', // "Beta"
	// Installation Guide
	'install.autoTab': 'स्वचालित CLI', // "Automatic CLI"
	'install.manualTab': 'मैन्युअल सेटअप', // "Manual Setup"
	// `<DeployGuidesNav>` vocabulary
	'deploy.sectionTitle': 'पृष्ठ संचालन मार्गदर्शिकाएँ', // "Deployment Guides"
	'deploy.altSectionTitle': 'अधिक पृष्ठ संचालन मार्गदर्शिकाएँ', // "More Deployment Guides"
	'deploy.filterLabel': 'डिप्लॉय प्रकार से छाँटें', // "Filter by deploy type"
	'deploy.ssrTag': 'SSR', // "SSR"
	'deploy.staticTag': 'Static', // "Static"
	// CMS Guides vocabulary
	'cms.navTitle': 'और CMS मार्गदर्शिकाएँ', // "More CMS guides"
	// Migration Guides vocabulary
	'migration.navTitle': 'और माइग्रेशन मार्गदर्शिकाएँ', // "More migration guides"
	// Recipes
});
```